packages/front-end/components/Features/ScheduleInputs.tsx

```diff
@@ -0,0 +1,180 @@
+import { ScheduleRule } from "back-end/types/feature";
+import clsx from "clsx";
+import React, { useEffect, useState } from "react";
+import Field from "../Forms/Field";
+import SelectField from "../Forms/SelectField";
+import styles from "./ScheduleInputs.module.scss";
+
+interface Props {
+  defaultValue: ScheduleRule[];
+  onChange: (value: ScheduleRule[]) => void;
+}
+
+const getLocalDateTime = (rawDateTime: string) => {
+  if (!rawDateTime) {
+    return "";
+  }
+  const utcDateTime = new Date(rawDateTime);
+
+  // We need to adjust for timezone/daylight savings time before converting to ISO String to pass into datetime-local field
+  utcDateTime.setHours(
+    utcDateTime.getHours() - new Date(rawDateTime).getTimezoneOffset() / 60
```

Comment:
Timezone offset is not always a multiple of 60. Some timezones are offset by 15 or 30 minutes, so set hours might not work properly. Timezones are really hard and there are so many edge cases. I highly recommend using a library instead of doing this yourself. We have the date-fns library loaded already if that helps.

Reply:
Thank you for suggesting `date-fns`. I've not used that previously but that was exactly what was needed in order to format the date in a way that the `datetime-local` input needed.